cg/meta/workflow/microsalt.py

```diff
@@ -164,7 +164,9 @@ def get_samples(self, case_id: str, sample_id: Optional[str] = None) -> List[Sam
         If sample_id is specified, will return a list with only this sample_id.
         Otherwise, returns all samples in given case"""
         if sample_id:
-            return [self.status_db.query(Sample).filter(Sample.internal_id == sample_id).first()]
+            return [
+                self.status_db.session.query(Sample).filter(Sample.internal_id == sample_id).first()
+            ]
 
         case_obj: Family = self.status_db.get_case_by_internal_id(internal_id=case_id)
         return [link.sample for link in case_obj.links]
```

Comment:
does this also require a .session?

Reply:
No

Reply:
It is only needed when we interact directly with the database